### PR TITLE
Change stocks rounding

### PR DESCRIPTION
**Pull request by insacuri**
## What? and Why?
The rounding in the spec was wrong.  Originally it was to the nearest thousand (56201 became 56000).  It's actually to the nearest thousand, then divided by 1000 (56201 becomes 56).

This required a small change to the transformer, and a lot of changes to the test data.

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ### Unreleased
  - Add calculated totals for QSI as the totals aren't passed to SDX
  - Fix missing date qcodes (11 and 12) for QSI
- - Add formtype 0070 for QSI (Stocks)
+\- Add formtype 0070 for QSI (Stocks)
```
Comment (necrophonic) [inline CHANGELOG.md]: Erroneous slash

File: transform/transformers/common_software/pck_transformer.py [hunk 2]
```diff
@@ -311,7 +311,7 @@ def round_numeric_values(self):
 
         if self.survey.get('survey_id') in [self.qsi_survey_id]:
             decimal.getcontext().rounding = ROUND_HALF_UP
-            self.data.update({k: str(Decimal(round(Decimal(float(v))) / 1000).quantize(1) * 1000)
+            self.data.update({k: str(self.round_to_nearest_thousand(v))
```
Comment (necrophonic) [inline transform/transformers/common_software/pck_transformer.py]: As `ROUND_HALF_UP` is the default operation for `round_to_nearest_thousand()` you don't need to specify it. I think that makes this call identical to the one below it so could just be combined?
```
if self.survey.get('survey_id') in [self.qcas_survey_id, self.qsi_survey_id]:
    self.data.update({k: str(self.round_to_nearest_thousand(
        for k, v in self.data.items() if k in self.qcas_currency_questions})
```

Comment (insacuri) [reply transform/transformers/common_software/pck_transformer.py]: Good spot

File: CHANGELOG.md (hunk 1 above)
Comment (insacuri) [reply CHANGELOG.md]: Indeed

File: transform/transformers/common_software/pck_transformer.py (hunk 2 above)
Comment (necrophonic) [reply transform/transformers/common_software/pck_transformer.py]: My bad, misread the end of the function so they're not exactly the same. Too many acronyms lead to confuzzle :)

**Pull request merged**